ivy/functional/frontends/torch/Tensor.py

```diff
@@ -126,7 +126,11 @@ def _to_with_device(
         return ivy.asarray(self.data, device=device, dtype=dtype, copy=copy)
 
     def arctan(self, *, out=None):
-        return torch_frontend.arctan(self, out=out)
+        return torch_frontend.atan(self, out=out)
+
+    def arctan_(self):
```

Comment:
Could you try adding this decorator to fix the failure? `@with_unsupported_dtypes({"1.11.0 and below": ("bfloat16")}, "torch")`